src/taipy/gui/server.py

```diff
@@ -180,7 +181,20 @@ def _is_port_open(self, host, port) -> bool:
         sock.close()
         return result == 0
 
+    def _apply_patch(self):
```

Comment:
Nice ! 
A way to detect if patching has been done

Reply:
Thanks!